www/js/app.js

```diff
@@ -49,6 +49,18 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
      */
     var selectedArchive = null;
     
+    /**
+     * A global parameter object for storing variables that need to be remembered between page loads
+     * or across different functions
+     * 
+     * @type Object
+     */
+    var params = {};
+
+    // Set parameters and associated UI elements from cookie
+    params['hideActiveContentWarning'] = /hideActiveContentWarning=true/i.test(document.cookie);
```

Comment:
Why don't you use the functions of cookies.js to read the cookie like you did line 203 to set it? Is it related to #438?

Reply:
@mossroy Actually, not directly related to #438 in this instance. It's just a very concise and fast way of setting the state of the paramater to true or false. Saves some CPU!! With cookies plugin it returns a text value from the cookie which I then have to test with a conditional to transform it to true/false. This was the most concise way of getting a Boolean. Happy to change if you don't mind the code expanding a bit.

Reply:
It's true that your implementation is probably faster.
On the other hand, I don't think we have performance issues regarding reading/setting cookies.
And this implementation might lead to some nasty bugs : if I set a cookie called "zzhideActiveContentWarning", it probably matches the regexp. Same thing if the value is "truezz".
In general, when there is no known performance issue on a feature, I prefer to centralize in a common implementation (even if it's a bit slower) : this lets us fix this kind of bugs in a single place, and also lets us easily refactor the code when necessary (like for #438).
So I'd prefer that you use the functions of cookies.js here

Reply:
`/\bhideActiveContentWarning=true\b/.test(document.cookie)` would mitigate against both of the attacks on this value that you mention. However, I think your point is more philosophical ;-) and about consistency, so I'm happy to change it to use cookie.js.